benchmark/scripts/generate_harness/generate_harness.py

```diff
@@ -12,43 +12,70 @@
 #
 # ===---------------------------------------------------------------------===//
 
-# Generate CMakeLists.txt and utils/main.swift from templates.
+# Generate boilerplate, CMakeLists.txt and utils/main.swift from templates.
 
 from __future__ import print_function
 
-import glob
+import argparse
+import jinja2
```

Comment:
@gottesmm Can you verify that the `lit` part of the harness test works like you wanted it to? I’m not quite sure how to invoke only that validation-test. (My 2008 MBP takes enough beating running the compilation and benchmarks as it is...)